js/src/personal/haoking/Haoking.module.css

```diff
@@ -1,4 +1,36 @@
-.title {
+.hkbody .animate_fade_out {
```

Comment:
Anything that is already a class doesn't need to be prefixed like this. It's only for like element selectors and such like div, p, a, etc.

We're using CSS Modules, and what they do is turn the class names into locally scoped class names by adding a hash to the class name. I don't think that applies to element selectors, which is why those should be prefixed with the hkbody class.

Reply:
ohh TIL

Reply:
Fixed on new commit